src/xarm_base_config/config/control/controllers.yaml

```diff
@@ -100,12 +100,9 @@ servo_controller:
 
 velocity_force_controller:
   ros__parameters:
-    planning_group_name: xarm7
+    planning_group_name: manipulator
```

Comment:
is this just a convenience/common practice here or is there a Pro requirement behind this?